server/src/main.rs

```diff
@@ -521,20 +534,21 @@ fn debug_execute(caps: Captures, _body: Launch, child: ChildThread) -> io::Resul
         .ok_or(io::Error::from(io::ErrorKind::NotConnected))?;
 
     child.tx.send(ServerMessage::Continue).unwrap();
-    let id = match child.rx.recv().unwrap() {
-        DebugMessage::Executing => child.next_id(),
+    match child.rx.recv().unwrap() {
+        DebugMessage::Executing => {},
         DebugMessage::Error(e) => return Err(e),
         _ => unreachable!(),
     };
-    child.execution = Some(id);
```

Comment:
This line still needs to be here so the server knows which execution is currently active.